tests/Agent/IntegrationTests/IntegrationTestHelpers/RemoteServiceFixtures/RemoteApplicationFixture.cs

```diff
@@ -584,66 +587,17 @@ protected void PostJson(string address, string payload, List<KeyValuePair<string
             Assert.True(result.IsSuccessStatusCode);
         }
 
-    }
-
-
-    // borrowed from XUnit.Sdk.ExecutionTimer, as using their implementation caused a runtime error looking for xunit.execution.dotnet.dll
-    /// <summary>
-    /// Measures and aggregates execution time of one or more actions.
-    /// </summary>
-    public class ExecutionTimer
-    {
-        TimeSpan total;
-
-        /// <summary>
-        /// Returns the total time aggregated across all the actions.
-        /// </summary>
-        public decimal Total
-        {
-            get { return (decimal)total.TotalSeconds; }
-        }
-
-        /// <summary>
-        /// Executes an action and aggregates its run time into the total.
-        /// </summary>
-        /// <param name="action">The action to measure.</param>
-        public void Aggregate(Action action)
-        {
-            var stopwatch = Stopwatch.StartNew();
-            try
-            {
-                action();
-            }
-            finally
-            {
-                total += stopwatch.Elapsed;
-            }
-        }
-
-        /// <summary>
-        /// Executes an asynchronous action and aggregates its run time into the total.
-        /// </summary>
-        /// <param name="asyncAction">The action to measure.</param>
-        public async Task AggregateAsync(Func<Task> asyncAction)
+        // Tests for things like transaction garbage collected, response time already captured, etc.
+        // Works best when logging is at FINEST.
+        private void TestForKnownProblems()
```

Comment:
How about making this `virtual` so it can be extended/overridden when necessary?

Reply:
Will do when I reimplement this in main.